index.js

```diff
@@ -129,7 +212,22 @@ function filterByGenre() {}
       }
     ];
  */
-function getAllMoviesReleasedAtOrBeforeYear() {}
+function getAllMoviesReleasedAtOrBeforeYear(movies, year) {
+  if(movies.length === 0){
+    return []
+  }
+
+  let arr = []
+
+  for (movie of movies){
+    let movieYear = Number(movie.released.slice(7))
+    if(movieYear <= year){
+      arr.push(movie)
+    }
+  }
```

Comment:
This technically works but be careful with the hardcoded `.slice()`.  What if we had a data set that could be written "Twenty two November, 1976"? 
This is an edge case but its something to consider even though our tests are not asking us to consider it,